packages/istanbul-lib-source-maps/test/map-store.test.js

```diff
@@ -102,8 +102,8 @@ describe('map store', function () {
             "column": 4
           },
           "end": {
-            "line": null,
-            "column": -1
+            "line": 2,
```

Comment:
similarly it seemed weird that `(anonymous_3)` had a different `decl` which includes the entire function body, as `decl` is supposed to only include the class declaration I believe, so: `function foo ()`.

This could be the fault of Istanbul though, and how code is transpiled?

```js
  // var SimpleClass = (function () {
  //     function SimpleClass() {
  //     }
  //     SimpleClass.prototype.hy = function () {
  //         console.log("Hy");
  //     };
  //     return SimpleClass;
  // }());
```